xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -62,6 +62,15 @@
 #include "view/GUIViewState.h"
 #include <inttypes.h>
 
+
+
+#include <pthread.h>
```

Comment:
don't forget about windows. there is no `pthread` support on windows, use STD
so many new lines? what for?